server/views/pages/add-id.njk

```diff
@@ -1,178 +1,60 @@
-                    <input class="govuk-input govuk-date-input__input govuk-input--width-2" id="id-expiry-date-day" name="id-expiry-date-day" type="text" inputmode="numeric" value="{{ data['id-expiry-date-day'] }}">
-                </div>
-            </div>
-            <div class="govuk-date-input__item">
-                <div class="govuk-form-group">
-                    <label class="govuk-label govuk-date-input__label" for="id-expiry-date-month">
-                        Month
-                    </label>
-                    <input class="govuk-input govuk-date-input__input govuk-input--width-2" id="id-expiry-date-month" name="id-expiry-date-month" type="text" inputmode="numeric" value="{{ data['id-expiry-date-month'] }}">
-                </div>
-            </div>
-            <div class="govuk-date-input__item">
-                <div class="govuk-form-group">
-                    <label class="govuk-label govuk-date-input__label" for="id-expiry-date-year">
-                        Year
-                    </label>
-                    <input class="govuk-input govuk-date-input__input govuk-input--width-4" id="id-expiry-date-year" name="id-expiry-date-year" type="text" inputmode="numeric" value="{{ data['id-expiry-date-year'] }}">
-                </div>
-            </div>
+    <div class="govuk-grid-row govuk-!-padding-top-4 govuk-!-padding-bottom-4">
+        {{ titleAndIntro(pathway) }}
+    </div>
+    <div class="govuk-grid-row govuk-!-padding-bottom-4">
+        <div class="govuk-grid-column-full">
+            <h2>Apply for an ID</h2>
+            <form action="id-add-2" class="govuk-form">
+                <div class="govuk-form-group">
+                    <label class="govuk-label" for="type">Type</label>
+                    <select class="govuk-select" id="type" name="type">
+                        <option value="">Select type</option>
+                        <option value="birth-certificate">Birth certificate</option>
+                        <option value="replacement-marriage-certificate">Replacement marriage certificate</option>
+                        <option value="replacement-civil-partnership-certificate">Replacement civil partnership certificate</option>
+                        <option value="adoption-certificate">Adoption certificate</option>
+                        <option value="divorce-decree-absolute-certificate">Divorce decree absolute certificate</option>
+                        <option value="driving-licence">Driving licence</option>
+                        <option value="biometric-residence-permit">Biometric residence permit</option>
+                        <option value="deed-poll-certificate">Deed poll certificate</option>
+                    </select>
+
+                </div>
+
+                <div class="govuk-form-group">
+                    <div class="govuk-date-input" id="date-requested">
+                        <label class="govuk-label" for="type">Application submitted</label>
```

Comment:
Perhaps replace the `for="type"` with `for="date-requested-day"` so that this label doesn't focus on the previous form element?

Reply:
Good shout